google/resource_compute_ssl_certificate.go

```diff
@@ -16,27 +33,49 @@ func resourceComputeSslCertificate() *schema.Resource {
 		Delete: resourceComputeSslCertificateDelete,
 
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: resourceComputeSslCertificateImport,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(240 * time.Second),
+			Delete: schema.DefaultTimeout(240 * time.Second),
 		},
 
 		Schema: map[string]*schema.Schema{
-			"certificate": &schema.Schema{
+			"certificate": {
 				Type:      schema.TypeString,
 				Required:  true,
 				ForceNew:  true,
 				Sensitive: true,
 			},
-
-			"name": &schema.Schema{
+			"private_key": {
+				Type:      schema.TypeString,
+				Required:  true,
+				ForceNew:  true,
+				Sensitive: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"name": {
 				Type:          schema.TypeString,
-				Optional:      true,
 				Computed:      true,
+				Optional:      true,
 				ForceNew:      true,
-				ConflictsWith: []string{"name_prefix"},
 				ValidateFunc:  validateGCPName,
+				ConflictsWith: []string{"name_prefix"},
 			},
-
-			"name_prefix": &schema.Schema{
+			"certificate_id": {
+				Type:     schema.TypeInt,
```

Comment:
Formerly a string - is that a problem?

Reply:
Nope - since this is `Computed` it gets magicked into a string anyways and won't change anything.

Reply:
Okay!

Reply:
Do make sure to note it in the changelog, though, of course.